Assets/HoloToolkit-Examples/Input/Scenes/XboxControllerExample.unity

```diff
@@ -180,9 +180,8 @@ Transform:
   m_LocalScale: {x: 1, y: 1, z: 1}
```

Comment:
Should this scene work when running in the editor? Hitting play and having a controller connected (I'm using it via USB) always shows `No Controller Connected`, even when I can teleport and such with the controller.

Reply:
Are you using an older Xbox or the Xbox one S?

Reply:
Xbox One S

Reply:
So the input still works even though it says no controller connected?

Reply:
I'm able to teleport, but I'm not able to do anything with the cube (should I be?)

Reply:
Interestingly, I do get a debug statement
![image](https://user-images.githubusercontent.com/3580640/33629686-cf6dfbc2-d9b9-11e7-8880-072d7108676b.png)

Reply:
Seems like the scene may have gotten messed up in a merge. I'll double check this stuff later today.

Reply:
Should be fixed now.